joern-cli/src/main/resources/scripts/c/malloc-leak.sc

```diff
@@ -0,0 +1,20 @@
+import io.shiftleft.codepropertygraph.Cpg
+import io.shiftleft.codepropertygraph.generated.nodes.Call
+import io.shiftleft.dataflowengine.language._
+import io.shiftleft.semanticcpg.language._
+
+private def mallocCalls(cpg: Cpg): Steps[Call] = {
```

Comment:
I can imagine that in the future, we'll want to more generically identify allocation calls, but let's get this in first :)